tests/ui/never_type/unused_trait_in_never_pattern_body.stderr

```diff
@@ -18,10 +18,18 @@ LL | |
 LL | |             use std::ops::Add;
 LL | |             0.add(1)
 LL | |         },
-   | |         ^
-   | |         |
-   | |_________this will never be executed
-   |           help: remove this expression
+   | |_________^ this will never be executed
+   |
+help: remove the match arm expression
+   |
+LL -         ! => || {
+LL -
+LL -
+LL -             use std::ops::Add;
+LL -             0.add(1)
+LL -         },
+LL +         !,
```

Comment:
This is a lot of lines, to the point that it's a bit hard to even see (without coloration) that this is even suggesting to leave the plain `!` and remove everything else. we should probably make this suggestion hidden?